packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -256,23 +256,23 @@ export class ApiReferencesTableSet extends React.Component<IApiReferencesTableSe
     const classMethods: IMethod[] = [];
 
     const members: ITableRowJson[] = table.members as ITableRowJson[];
-    for (let k = 0; k < members.length; k++) {
-      if (members[k].kind === 'Method') {
+    members.map((member: ITableRowJson) => {
```

Comment:
You should use `forEach` (or `for..of`) since you're not returning a value:
```suggestion
    members.forEach((member: ITableRowJson) => {
```